packages/libs/react-ui/src/components/Breadcrumbs/index.tsx

```diff
@@ -1,17 +1,4 @@
-import type { FC } from 'react';
-import type { IBreadcrumbsProps } from './Breadcrumbs';
-import { BreadcrumbsContainer } from './Breadcrumbs';
-import type { IBreadcrumbItemProps } from './BreadcrumbsItem';
-import { BreadcrumbsItem } from './BreadcrumbsItem';
-
-interface IBreadcrumbs {
-  Root: FC<IBreadcrumbsProps>;
-  Item: FC<IBreadcrumbItemProps>;
-}
-
-export type { IBreadcrumbItemProps, IBreadcrumbsProps };
-
-export const Breadcrumbs: IBreadcrumbs = {
-  Root: BreadcrumbsContainer,
-  Item: BreadcrumbsItem,
-};
+export { BreadcrumbsContainer } from './Breadcrumbs';
```

Comment:
In other components, the "container" component is just named what the component is. So in this case I think it would be better if this was just exported as `Breadcrumbs`

so when used it would look like this: 
```
<Breadcrumbs>
  <BreadcrumbItem>Item<BreadcrumbItem/>
</Breadcrumbs>
```

Reply:
I like this indeed will add the change